scripts/release.py

```diff
@@ -0,0 +1,221 @@
+import os
+import re
+import subprocess
+
+from github import Github
+
+
+"""This release notes script is built to create a release notes draft 
+for release candidates, patches, and minor releases.
+
+Setup:
+1.Create Github token: 
+https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic # noqa
+2. Give the Github token repo, user, audit_log, and project permissions.
+3. Add `export GH_TOKEN=<github token>` to the `.zhrc` file.
+
+Usage:
+Required:
+    BASE - The base branch you are building your release candidate, patch, or minor release off of.
+        If this is a rc1, then just specify the branch you'll create after the release is published. e.g. BASE=1.9
+
+Optional:
+    RC - Whether or not this is a release candidate. e.g. RC=1 or RC=0
+    PATCH - Whether or not this a patch release. e.g. PATCH=1 or PATCH=0
+    PRINT - Whether or not the release notes should be printed to CLI or be used to create a Github release. Default is 0 e.g. PRINT=1 or PRINT=0    
```

Comment:
Yeah PRINT=0 will create a draft release, but it defaults to 0, so basically in my mind the norm will be to just not add the print envar to your command, because you want to generate release notes, not print them. If you do want to print them, then the option is there.